include/nle.h

```diff
@@ -21,6 +23,10 @@ typedef struct nle_globals {
     char *outbuf_write_ptr;
     char *outbuf_write_end;
 
+#ifdef NLE_BZ2_TTYRECS
+    void *ttyrec_bz2;
+#endif
+
```

Comment:
```suggestion
#ifdef NLE_BZ2_TTYRECS
    void *ttyrec_bz2;
#else 
    FILE *ttyrec;
#endif

```

I'm curious: why aren't we doing ^^^ ? We never write to the file if the macro is set. Do we not just run the risk of confusion by keeping this pointer around?

Reply:
Good question. Who exactly is responsible for this `FILE` is a bit of a tricky question. For now, I'd like to keep it around for debugging and to allow flushing (although we don't do that currently).